app/models/draft.rb

```diff
@@ -237,19 +237,18 @@ def self.stream(opts = nil)
   end
 
   def self.cleanup!
-    DB.exec(<<~SQL)
-      DELETE FROM drafts
-       WHERE sequence < (
+    Draft.where(<<~SQL).in_batches(of: 100).destroy_all
```

Comment:
Ultra NIT, what do you think about extracting 100 to constant to avoid magic number?

Reply:
I normally do this, and considered it in this case too, but decided not to in this case for two reasons:

1. It is a somewhat arbitrary number in this case. Any name we can give it wouldn't be any more meaningful than inlining it.
2. It would be hoisted far away instead of being co-located with its use, which became the tipping point for me to not move it.

🙂

Reply:
Thank you for your explanation. I think that 100 was a number out of the air so maybe in the future we should just call just in_batches and relay on they 1000 default 🤷‍♂️